evennia/web/website/templates/website/_menu.html

```diff
@@ -0,0 +1,60 @@
+{% comment %}
+Allow to customize the menu that appears at the top of every Evennia
+webpage. You could copy this file in your mygame's web/template_overrides/website
```

Comment:
Suggested change of phrase: "Copy this file to your game dir's `web/template/overrides/website` folder and edit it there."